views/default/js/mentions/editors/tinymce.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Mentions support for the TinyMCE editor
+ */
+define(function(require) {
+	var mentions = require('mentions/autocomplete');
+
+	// Give some time for the TinyMCE to load
```

Comment:
There might be. AFAIK back when this was made, TinyMCE was not available as AMD module so this was the only way. I'm not sure of the current state.